packages/block-library/src/paragraph/edit.js

```diff
@@ -195,6 +195,7 @@ class ParagraphBlock extends Component {
 					onRemove={ onReplace ? () => onReplace( [] ) : undefined }
 					aria-label={ content ? __( 'Paragraph block' ) : __( 'Empty block; start writing or type forward slash to choose a block' ) }
 					placeholder={ placeholder || __( 'Start writing or type / to choose a block' ) }
+					__unstableEmbedURLOnPaste
```

Comment:
Is this prop newly introduced in this PR? (Could not find its reference anywhere else in the codebase.) What is its purpose?

Not sure, what's the convention for such unstable new props. But can we add some doc on this? Maybe inline comment, readme, somewhere else? Maybe better prop naming to reflect its use case better?

Reply:
I can provide an inline comment, but I thought the name is quite clear: it enables embedding when the user pastes a URL. This prop should normally only be used on the core paragraph block. We shouldn't document unstable props in a readme file.

> An unstable API is one which serves as a means to an end. It is not desired to ever be converted into a public API.

https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/